test/suites/common/run-test.yaml

```diff
@@ -25,8 +29,16 @@ spec:
     workingDir: $(workspaces.ws.path)
     script: |
       echo "Cloning $(params.git-repo-url)"
+      echo "Fetching - Optional: $(params.git-fetch-origin)"
       echo "Checking out git ref: $(params.git-ref)"
-      git clone $(params.git-repo-url).git .
+
+      git clone $(params.git-repo-url).git
+
+      if [ "$(params.git-fetch-origin)" != "" ]; then
```

Comment:
I don't think we need this parameter. Why not just fetch all the time and build the fetch command from the git-ref param?  

You could also just expect the `pull/ID/head:#1234` to be the input to git-ref if you want a PR and then `git fetch ${params.git-ref}` all the time.

Reply:
done good idea